### PR TITLE
Ab photogram industrial 2

### PR DESCRIPTION
**Pull request by aborov**

### CODE REVIEW

**Review by jniewiel — APPROVED** [review]
Approved.

**Review by aizatibraimova — APPROVED** [review]
Looks good

**Review by jniewiel — APPROVED** [review]
Looks good.

**Review by aizatibraimova — APPROVED** [review]

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#  index_likes_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
```
Comment (thierrychau) [inline app/models/like.rb:23]: Can you add a validation to prevent a user to like the same photo more than once?

File: public/example_dev.rake
Comment (thierrychau) [inline public/example_dev.rake:1]: Any reason why there are two sample data methods? This one and lib/tasks/dev.rake

**Review by thierrychau — CHANGES_REQUESTED** [review]
Please see comments below

File: public/example_dev.rake
Comment (aborov) [reply public/example_dev.rake:1]: This one is called example_sample_data, so it's not being run by `rake sample_data`, but thanks for the catch. I would like to keep it around for reference, so I just commented it out to be safe.

File: app/models/like.rb (hunk 1 above)
Comment (aborov) [reply app/models/like.rb:23]: Sure! I just did with the recent commit.

**Review by thierrychau — APPROVED** [review]
Looks good, you can merge it. 

For future reference: let the reviewer resolve conversations

**Pull request merged**